bundle/manifests/trustee-operator.clusterserviceversion.yaml

```diff
@@ -19,18 +19,15 @@ metadata:
             "namespace": "kbs-operator-system"
           },
           "spec": {
-            "kbsAsConfigMapName": "as-config-grpc",
             "kbsAuthSecretName": "kbs-auth-public-key",
-            "kbsConfigMapName": "kbs-config-grpc",
-            "kbsDeploymentType": "MicroservicesDeployment",
-            "kbsRvpsConfigMapName": "rvps-config-grpc",
-            "kbsRvpsRefValuesConfigMapName": "rvps-reference-values",
-            "kbsServiceType": "ClusterIP"
+            "kbsConfigMapName": "kbs-config",
+            "kbsDeploymentType": "AllInOneDeployment",
```

Comment:
So the default deployment becomes monolithic with this change?

Reply:
Yes, now the monolithic kbs-config-sample config map is created automatically and ready to use as soon as the KbsConfig CRD is created.